docs/content/patterns/crud-data-tables/index.mdx

```diff
@@ -0,0 +1,448 @@
+			<Table>
+				<TableHead>
+					<TableRow>
+						<TableHeader scope="col">Item</TableHeader>
+						<TableHeader scope="col">Qty</TableHeader>
+						<TableHeader scope="col">Files</TableHeader>
+						<TableHeader scope="col">Actions</TableHeader>
+					</TableRow>
+				</TableHead>
+				<TableBody>
+					<TableRow>
+						<TableCell>Type 1</TableCell>
+						<TableCell>1</TableCell>
+						<TableCell as="th" scope="row">
+							FileName1.png
+						</TableCell>
+						<TableCell>
+							<TextLink aria-describedby="FileName1.png" href="#">
+								Remove
+							</TextLink>
+						</TableCell>
+					</TableRow>
+				</TableBody>
+			</Table>
+		</TableWrapper>
+		<ButtonGroup>
+			<Button variant="secondary" size="sm" iconBefore={PlusIcon}>
+				Upload file
+			</Button>
+		</ButtonGroup>
+	</Stack>
+</Stack>
+```
+
+If a required file is not uploaded before the page is submitted, show an error in the row where the file is missing and provide a section alert above the table to assist with error correction.
+
+## Upload a defined list of files to a data table
+
+A specific or defined list of files can be uploaded several ways. The number of items on the list and number of files for each list item will determine the appropriate pattern to use.
+
+The upload defined list of files pattern reduces the amount of form content on the parent page making it easier for users to complete the form, and keeps the users context and focus when providing all required metadata in the side drawer or subpage.
+
+### Empty state
+
+Before any files are uploaded provide an empty state. We prefer using a standard empty state rather than an empty table because the experience of the empty state will be better for screen reader users. They will not need to navigate through a table to find out it is empty.
+
+```jsx live
+<Stack gap={3}>
+	<Stack gap={1.5}>
+		<H1>Upload defined list of files to data table</H1>
+	</Stack>
+	<Stack gap={2} alignItems="flex-start">
+		<H2>List of equipment</H2>
+		<Stack gap={1}>
+			<InfoIcon size="lg" />
+			<Heading type="h2" fontSize="lg">
+				No equipment added
+			</Heading>
+			<Text>You must add an equipment to proceed.</Text>
+		</Stack>
+		<ButtonGroup>
+			<Button variant="secondary" size="sm" iconBefore={PlusIcon}>
+				Upload file
+			</Button>
+		</ButtonGroup>
+	</Stack>
+</Stack>
+```
+
+### Large lists
+
+When large defined lists of files in tables are required a drawer or subpage for each upload action should be used. The File Upload component can be used in the side drawer or subpage to allow multiple files for each required category to listed and uploaded to a table.
+
+### File categorisation
+
+The use of a subpage or drawer enables you to insert questions and fields specific to each file upload to define the category and provide any additional information or metadata related to an upload. Radio groups or Selects can be used to help the user determine which category should be applied to an uploaded file.
```

Comment:
```suggestion
The use of a subpage or drawer enables you to insert questions and fields specific to each file upload to define the record category or type, and provide any additional information related to an upload. Radio groups or selects can be used to help the user determine which category should be applied to an uploaded file.
```

Reply:
this is good but Im worried that designers and the business both use the term metadata

Reply:
I was consolidating it with the other one. metadata instead of additional information works, too